app/controllers/Support.scala

```diff
@@ -37,6 +37,21 @@ class Support(
 
   private val httpClient = new OkHttpClient.Builder().connectTimeout(5, TimeUnit.SECONDS).build
 
+  def checkFileExtension(image: File, filename: String): Either[String, File] = {
+    val fileExtension = filename.contains(".") match {
+      case true => filename.substring(filename.lastIndexOf("."));
+      case false => ""
+    }
```

Comment:
[nit] pattern match is overkill for booleans, you can inline `if` in scala
```suggestion
    val fileExtension = if(filename.contains(".")) filename.substring(filename.lastIndexOf(".")) else ""
    }
```

Reply:
Good suggestion, have updated now.